frontend/src/components/AddIngredients.js

```diff
@@ -1,31 +1,47 @@
 import styled from "styled-components/macro";
+import { TextField } from "@material-ui/core";
+import { Autocomplete } from "@material-ui/lab";
 import { useState } from "react";
 
-export default function AddIngredients({ onAddClick }) {
+export default function AddIngredients({ ingredients, setIngredientsList }) {
   const [ingredient, setIngredient] = useState("");
 
-  const handleAddClick = () => {
-    onAddClick(ingredient);
-    setIngredient("");
+  const handleAddClick = (event) => {
+    event.preventDefault();
+    setIngredientsList((ingredientsList) => [
+      ...ingredientsList,
+      ingredient.name,
+    ]);
+  };
+
+  const handleInputChange = (event, value) => {
+    if (value !== null) {
+      setIngredient(value);
+    }
   };
 
   return (
-    <Wrapper>
-      <input
-        type="text"
-        placeholder="Enter Ingredient"
-        value={ingredient}
-        onChange={(event) => setIngredient(event.target.value)}
+    <Wrapper onSubmit={handleAddClick}>
```

Comment:
makes it easier and faster to identify which kind of component this is

```suggestion
    <Form onSubmit={handleAddClick}>
```